templates/_helpers.tpl

```diff
@@ -103,6 +103,11 @@ extra volumes the user may have specified (such as a secret with TLS).
   {{- if .Values.server.volumes }}
     {{- toYaml .Values.server.volumes | nindent 8}}
   {{- end }}
+  {{- if (and .Values.server.enterpriseLicense.secretName .Values.server.enterpriseLicense.secretKey) }}
+        - name: vault-license
+          secret:
+            secretName: {{ .Values.server.enterpriseLicense.secretName }}
```

Comment:
Nit: should we set the perms on this file?

Reply:
Sure, something like `defaultMode: 0440`?

Reply:
Works for me!

Reply:
Done in 95beeccfd6162ccf80a6bf97e64338fc5238f2f1